source/patterns/@aws-solutions-constructs/core/lib/lambda-event-source-mapping-defaults.ts

```diff
@@ -47,4 +40,17 @@ export function S3EventSourceProps(_s3EventSourceProps?: S3EventSourceProps) {
     } else {
         return defaultS3EventSourceProps;
     }
-}
\ No newline at end of file
+}
+
+export function KinesisEventSourceProps(_kinesisEventSourceProps?: KinesisEventSourceProps) {
+    const defaultKinesisEventSourceProps = {
```

Comment:
The reasoning behind these defaults was:
- TRIM_HORIZON: Same as DynamoDB stream default
- Bisect on error: This is a recent feature added by Lambda (https://aws.amazon.com/about-aws/whats-new/2019/11/aws-lambda-supports-failure-handling-features-for-kinesis-and-dynamodb-event-sources/), and enabling it _"breaks the impacted batch of records into two when a function returns an error, and retries them separately. This allows you to easily separate the malformed data record from the rest of the batch, and process the rest of data records successfully"_